Code/Framework/AzCore/AzCore/Serialization/EditContextConstants.inl

```diff
@@ -196,8 +196,19 @@ namespace AZ
             //! For use with slice creation tools. See SliceCreationFlags below for details.
             const static AZ::Crc32 SliceFlags = AZ_CRC("SliceFlags", 0xa447e1fb);
 
-            //! Does the clear button in the LineEdit need to have a test for visibility.
-            const static AZ::Crc32 ShowClearButtonHandler = AZ_CRC_CE("ShowClearButtonHandler");
+            //! On controls with a clear button, this attribute specifies whether the clear button should be shown.
+            const static AZ::Crc32 ShowClearButtonHandler = AZ_CRC("ShowClearButtonHandler", 0x2ef414c9);
```

Comment:
Any reason to stop using AZ_CRC_CE?

Reply:
I figure I'd use AZ_CRC for as long as I can since I'm tired of slow compile times and ConstExpr CRC ain't free. :)